### PR TITLE
add license metadata to package setup

### PR DESCRIPTION
**Pull request by jimthompson5802**
add license metadata to package setup definition

package scanning system prevents install of packages with missing license metadata. Unable to test this package.

### CODE REVIEW

**Review by Yard1 — APPROVED** [review]

**Comment by jimthompson5802** [comment]
@Yard1   Thank you for the approval.  Can I assume someone on the project will merge the PR when the project publishes the next release?

**Comment by Yard1** [comment]
Was just waiting for CI to pass :) We can make a new pip release this week.

**Pull request merged**
